slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -512,30 +531,43 @@ def onClickDeepgrow(self, current_point):
         if in_file is None and session_id is None:
             return
 
+        # use model name to determine
+        deepgrow_3d = True if '3d' in model.lower() else False
```

Comment:
this assumption needs to be documented somewhere.. against deepgrow feature.. 

https://github.com/NVIDIA/ai-assisted-annotation-client/tree/master/slicer-plugin#deepgrow-based-segmentation-of-liver-on-ct

we should fix this section and add note for 3D (above assumption)

Reply:
will do.